doc/index.md

```diff
@@ -113,4 +113,41 @@ public class Connection
         var myResult = await rpc.InvokeWithParameterObjectAsync<string>("test/InvokeTestMethod");
     }
 }
+```
+
+## Crashing the process on exception
+In some cases, you may want to immediately crash the server and client processes if certain exceptions are thrown. In this case, overriding the `IsFatalException` method will give you the desired functionality. `IsFatalException` provides an exception filter through which you can access and respond to exceptions as they are caught.
+```csharp
+public class Server : BaseClass
+{
+    public void ThrowsException() => throw new Exception("Throwing an exception");
+}
+
+public class JsonRpcCrashesOnException : JsonRpc
+{
+    public JsonRpcCrashesOnException(Stream clientStream, Stream serverStream, object target = null) : base(clientSteam, serverStream, target)
+    {
+    }
+
+    protected override bool IsFatalException(Exception ex)
+    {
+        if (ex.GetType() != typeof(OperationCanceledException))
```

Comment:
Prefer `ex is OperationCanceledException` over explicitly calling `GetType()` and `typeof`. In your case you'd need to flip the logic though, so either `!(ex is OperationCanceledException)` or flip the blocks around.